pytest.ini

```diff
@@ -9,3 +9,6 @@ filterwarnings =
 addopts = -v
 markers =
     skip_clickhouse_client: marks tests as requiring clickhouse client to be skipped
+    weave_client: alias for `client` fixture used for CI dispatching. Do not use directly in code.
```

Comment:
Sorry, where/how is this used?  It's not obvious to me

Reply:
in CI: `-m "weave_client and not skip_clickhouse_client" \`, and in code:

```
def pytest_collection_modifyitems(config, items):
    # Add the weave_client marker to all tests that have a client fixture
    for item in items:
        if "client" in item.fixturenames:
            item.add_marker(pytest.mark.weave_client)
```